webui/react/src/components/Metadata/EditableRow.tsx

```diff
@@ -55,9 +55,11 @@ const EditableRow: React.FC<Props> = ({ name, onDelete, field }: Props) => {
             getPopupContainer={(triggerNode) => triggerNode}
             menu={menu}
             trigger={['click']}>
-            <Button aria-label="action" ghost type="text">
-              <Icon name="overflow-vertical" size="tiny" />
-            </Button>
+            <Button
+              aria-label="action"
+              ghost
+              icon={<MoreOutlined style={{ fontSize: '20px' }} />}
```

Comment:
@keita-determined Why the change to `MoreOutlined` with styling instead of:


```
<Button
 icon={<Icon name="overflow-vertical" size="tiny" />}
 </Button>
```

?

Reply:
thats why PR needs to be reviewed. you are right. i was thinking about something else.
i'll replace them all with `<Icon />`

Reply:
fixed